memory-viz/src/typeguards.ts

```diff
@@ -25,6 +25,8 @@ export function isStyle(value: any): value is Style {
         "box_id",
         "box_type",
         "box_container",
+        "compound_text",
```

Comment:
Another change: to be consistent with the other attributes, let's go with `text_compound` and `box_compound`